test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,7 +90,7 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+        //assertEquals(addressBook, saveFile.load());
```

Comment:
Commented codes shouldn't be inside the commit; you have revision control to keep past important codes.